cloudinit/config/cc_ubuntu_advantage.py

```diff
@@ -383,12 +386,21 @@ def _auto_attach(ua_section: dict):
         LOG.error(msg)
         raise RuntimeError(msg) from ex
 
+    enable = ua_section.get("enable")
+    enable_beta = ua_section.get("enable_beta")
     options = FullAutoAttachOptions(
-        enable=ua_section.get("enable"),
-        enable_beta=ua_section.get("enable_beta"),
+        enable=enable,
+        enable_beta=enable_beta,
     )
     try:
         full_auto_attach(options=options)
+    except AlreadyAttachedError:
+        if enable_beta is None or enable is None:
+            # Only warn if the user defined some service to enable/disable.
```

Comment:
I think for the condition to match the comment, this should be
```suggestion
        if enable_beta is not None or enable is not None:
            # Only warn if the user defined some service to enable/disable.
```

Reply:
Oopsie, fixed.